sql/core/benchmarks/MakeDateTimeBenchmark-results.txt

```diff
@@ -2,21 +2,21 @@ OpenJDK 64-Bit Server VM 1.8.0_252-8u252-b09-1~18.04-b09 on Linux 4.15.0-1063-aw
 Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
 make_date():                              Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-prepare make_date()                                2951           3182         355         33.9          29.5       1.0X
-make_date(2019, 9, 16)                             2325           2415         101         43.0          23.2       1.3X
-make_date(*, *, *)                                 4556           4573          17         21.9          45.6       0.6X
+prepare make_date()                                3309           3429         110         30.2          33.1       1.0X
```

Comment:
The formatting changes are positive of course. Most of the numbers didn't change much at all; these kinda of did, some much more than a few stdevs. Just checking, it was on the same hardware as in the heading?

Reply:
I think so, as the head is written by the benchmark framework automatically.